src/llamafactory/extras/constants.py

```diff
@@ -1152,6 +1152,18 @@ def register_model_group(
 )
 
 
+register_model_group(
+    models={
+        "Pixtral-12B": {
```

Comment:
model key should end with -Chat or -Instruct

Otherwise it will be recognized by webui as a pretrain model and not a chat model